test/main/deepEqual.js

```diff
@@ -1581,6 +1581,35 @@ QUnit.test( "Test that must be done at the end because they extend some primitiv
 	}
 );
 
+QUnit.test( "Compare primitive objects", function( assert ) {
+	var SafeNumber = Number,
+	SafeString = String,
+	SafeBoolean = Boolean;
+
+	assert.equal( QUnit.equiv(new SafeNumber(1), new SafeNumber(1) ), true,
+		"Number: When create same parameter should be equal."
+	);
```

Comment:
We may also simplify these tests using the ok and notOk assertions and/or using a single line for each parameter:

```js
assert.equal(
	QUnit.equiv( new SafeNumber(1), new SafeNumber(1) ),
	true,
	"Number: When create same parameter should be equal."
);

// or simply:
assert.ok(
	QUnit.equiv( new SafeNumber( 1 ), new SafeNumber( 1 ) ),
	"Number: When create same parameter should be equal."
);
```